src/test/java/org/snomed/snowstorm/core/rf2/export/ExportServiceTest.java

```diff
@@ -482,7 +486,46 @@ void exportRF2ArchiveForClassification() throws Exception {
 			assertEquals(owlMember.getId() + "\t\t1\t900000000000207008\t733073007\t123005000\tTransitiveObjectProperty(:123005000)", lines.get(1));
 		}
 	}
+	
+	@Test
+	void exportRF2ArchiveForRefsetOnly() throws Exception {
+		File exportFile = getTempFile("export", ".zip");
+		exportFile.deleteOnExit();
 
+		// Run export
+		try (FileOutputStream outputStream = new FileOutputStream(exportFile)) {
+			ExportConfiguration exportConfiguration = new ExportConfiguration("MAIN", RF2Type.SNAPSHOT);
+
+			// FOR Refsets only
+			Set<String> refsetIds = new HashSet<>();
+			refsetIds.add("900000000000508004");
+			exportConfiguration.setRefsetIds(refsetIds);
+			exportConfiguration.setConceptsAndRelationshipsOnly(false);
+			exportConfiguration.setLegacyZipNaming(false);
+			exportConfiguration.setUnpromotedChangesOnly(false);
+
+			exportConfiguration.setFilenameEffectiveDate("20210731");
+			exportService.createJob(exportConfiguration);
+			exportService.exportRF2Archive(exportConfiguration, outputStream);
+		}
+		
+		String descriptionTypeRefsetMemberId = referenceSetMemberService.findMembers("MAIN", Concepts.FSN, PAGE_OF_ONE).getContent().get(0).getMemberId();
+		String descriptionLanguageRefsetMemberId = referenceSetMemberService.findMembers("MAIN", descriptionId, PAGE_OF_ONE).getContent().get(0).getMemberId();
+		String textDefLanguageRefsetMemberId = referenceSetMemberService.findMembers("MAIN", textDefId, PAGE_OF_ONE).getContent().get(0).getMemberId();
+
+
+		// Test export
+		try (ZipInputStream zipInputStream = new ZipInputStream(new FileInputStream(exportFile))) {
+
+			ZipEntry axioms = zipInputStream.getNextEntry();
```

Comment:
Calling `getNextEntry` again after reading this one would prove that nothing else was exported. (Also this `axioms` variable has a misleading name.)